tests/merge/merge_helpers.c

```diff
@@ -40,7 +40,7 @@ int merge_trees_from_branches(
 	cl_git_pass(git_commit_tree(&our_tree, our_commit));
 	cl_git_pass(git_commit_tree(&their_tree, their_commit));
 
-	cl_git_pass(git_merge_trees(index, repo, ancestor_tree, our_tree, their_tree, opts));
+	error = git_merge_trees(index, repo, ancestor_tree, our_tree, their_tree, opts);
```

Comment:
Would this change (and the equivalent below) make the other users of `merge_trees_from_branches()` and `_commits_` to stop making sure that the merge succeeded? Other users should be changed to wrap their call in `cl_git_pass()` since we don't anymore, no?